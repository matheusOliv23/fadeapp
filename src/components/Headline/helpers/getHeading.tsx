import { HeadlineTitle } from '../styles'

export default function getHeading(as: string, title: string) {
  const classNames = `${HeadlineTitle}} `
  switch (as) {
    case 'h1':
      //prettier-ignore
      return <h1 className={classNames}>{title}</h1>
    case 'h2':
      //prettier-ignore
      return <h2 className={classNames}>{title}</h2>
    case 'h3':
      //prettier-ignore
      return <h3 className={classNames}>{title}</h3>
    case 'h4':
      //prettier-ignore
      return <h4 className={classNames}>{title}</h4>
    case 'h5':
      //prettier-ignore
      return <h5 className={classNames}>{title}</h5>
    case 'h6':
      //prettier-ignore
      return <h6 className={classNames}>{title}</h6>
  }
}
