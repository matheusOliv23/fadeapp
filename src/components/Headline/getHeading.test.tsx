import getHeading from './helpers/getHeading'

describe('getHeading', () => {
  it('should match snapshot', () => {
    const mockHeading = getHeading('h2', 'Teste')

    expect(mockHeading).toMatchSnapshot('h2')
    expect(mockHeading).toMatchSnapshot('Teste')
  })
})
