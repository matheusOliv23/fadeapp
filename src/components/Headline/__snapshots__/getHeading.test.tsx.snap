// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getHeading should match snapshot: Teste 1`] = `
<h2
  className="
  text-2xl font-bold sm:text-6xl    
} "
>
  Teste
</h2>
`;

exports[`getHeading should match snapshot: h2 1`] = `
<h2
  className="
  text-2xl font-bold sm:text-6xl    
} "
>
  Teste
</h2>
`;
